### ISSUE TITLE
About trainning acc

### ISSUE THREAD

**Issue by meanmee** [deepinsight/insightface#157]
I can just get 93% training acc after 90 epochs when train resnet50 with arc loss，could you tell me your final training acc while get 98% in megaface test？
Thank U
@nttstar

**Comment by nttstar** [comment]
Check the training log which was already included in pre-trained model.

**Comment by meanmee** [comment]
The training los is incomplete，There is only like 20+ epoch‘s training log in the log file

**Comment by meanmee** [comment]
seems like the training the model like 30 epochs with about 90% training acc is enough? Because after 136 epochs training with 95% training acc, the eval results in my own eval dataset is worse.
